scripts/bert/deploy.py

```diff
@@ -321,17 +321,29 @@ def export(prefix):
         arg_array['data2'] = mx.nd.ones((test_batch_size, ), dtype='float32')
```

Comment:
@Kh4L didnt you make a change in 1.8 that enables not having to set the inputs like this?
@MoisesHer is setting the shapes/types still not working when using `shape_dict`?

Reply:
yeap, If I set shapes/types, I am receiving a `Segmentation fault: 11` as soon as I call `sym.optimize_for`